static/app/components/onboarding/gettingStartedDoc/deprecatedPlatformInfo.tsx

```diff
@@ -0,0 +1,63 @@
+import styled from '@emotion/styled';
+
+import {CodeSnippet} from 'sentry/components/codeSnippet';
+import {Alert} from 'sentry/components/core/alert';
+import {Stack} from 'sentry/components/core/layout';
+import {ExternalLink} from 'sentry/components/core/link';
+import {Text} from 'sentry/components/core/text';
+import {t, tct} from 'sentry/locale';
+import type {PlatformIntegration, ProjectKey} from 'sentry/types/project';
+
+type DeprecatedPlatformInfoProps = {
+  dsn: ProjectKey['dsn'];
+  platform: PlatformIntegration;
+};
+
+export function DeprecatedPlatformInfo({platform, dsn}: DeprecatedPlatformInfoProps) {
+  return (
+    <NoOverflowAlert type="warning" showIcon={false}>
+      <Stack padding="xl" gap="xl">
+        <Text>
+          {tct(
+            '[platform] has been deprecated, but you can still use this project with the following DSN key:',
```

Comment:
Not sure if `DSN key` is the right terminology 🤔 
```suggestion
            '[platform] has been deprecated, but you can still use this project with the following DSN:',
```

Reply:
agree.